cmd/wallet_delete.go

```diff
@@ -17,11 +17,14 @@ var (
 	ErrForceFlagIsRequiredWithoutTTY = errors.New("--force is required without TTY")
 
 	deleteWalletLong = cli.LongDesc(`
-		Delete a given wallet.
+		Delete the specified wallet and its keys.
 
 		Be sure to have a backup of its mnemonic, otherwise you won't be able to restore
 		it. If you lost it, you should transfer anything attached to this wallet
 		to another wallet.
```

Comment:
This would be a good place to give people some help on how to transfer keys to another wallet.

Reply:
I wasn't referring transferring keys, but transferring funds, assets, orders... things like this. How ca we improve this, without adding a tutorial for console in the description ?

Reply:
I see! How about "transfer assets, or anything else attached..."

Reply:
Alright !